Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -173,10 +179,12 @@ public class RestRequest: NSObject  {
     ///
     /// - Parameters:
     ///   - url: URL string to use for network request
-    public init(method: HTTPMethod = .get, url: String, containsSelfSignedCert: Bool? = false) {
+    public init(method: HTTPMethod = .get, url: String, containsSelfSignedCert: Bool? = false, certificateName: String? = nil, certificatePath: String? = nil) {
```

Comment:
Do you think it would be better to define a

```
public struct clientCertificate {
    let name: String
    let path: String
}
```

and pass one of those in here?